vrk-kpa/suomidigi
# SUOPA-429 Community

To test: 
- `make fresh`
- Log in and go to http://suomidigi.fi.docker.amazee.io/admin/structure/taxonomy/manage/community_domain/overview 
   - Create few different domains (you may leave the image field empty - it's taken care of...)
- Go to http://suomidigi.fi.docker.amazee.io/admin/content/community
   - Click Add community, fill in the fields and save the entity - rinse and repeat
- Go to http://suomidigi.fi.docker.amazee.io/node/add/landing_page
   - Give the landing page a title, ingress and add new paragraph View
      - Select the only view in the list and the only display mode from the list
   - Save the node and see that the community list looks like this one: https://ixonos.invisionapp.com/d/main#/console/16859411/349551748/inspect
